tests/pd/test_rpc_client.rs

```diff
@@ -11,23 +11,28 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use std::env;
+use std::thread;
+use std::sync::Arc;
+use std::time::Duration;
 
 use kvproto::metapb;
 
-use tikv::pd::{PdClient, RpcClient};
+use tikv::pd::{PdClient, RpcClient, validate_endpoints};
+
+use super::mock::case::*;
+use super::mock::Server as MockServer;
 
 #[test]
 fn test_rpc_client() {
-    // We need to set all members in PD_ENDPOINTS to pass this test.
-    let endpoints = match env::var("PD_ENDPOINTS") {
-        Ok(v) => v,
-        Err(_) => return,
-    };
+    let eps = "http://127.0.0.1:3079".to_owned();
```

Comment:
`TEST_THREADS` in Cicle CI is 1, in Travis is 2, so conflict may occurs in Travis, since an IP have tens of thousands ports and we have hundreds of tests, I think the possibility is very low.